topics/admin/tutorials/ansible/slides.html

```diff
@@ -154,7 +156,7 @@
 
 ???
 
-**Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server.
+**Tasks** will be executed which will usually include tasks to **copy** files from the `files` directory and **template** files from the `templates` directory over to the remote server. Templates have a .j2 suffix, which indicates that they are Jinja 2 templates. After filling the template with the variable values, we copy the file to its remote destination without the .j2 suffix.
```

Comment:
I'd probably put this explanation in `### {% icon details %} Template Syntax` in the tutorial, where there is an example.

Reply:
Yep, did that.